src/ui/browserAction/popupPage.js

```diff
@@ -350,6 +367,50 @@ export class BrowserActionPopup extends LitElement {
       slot="left"
     ></mz-iconlink>`;
   }
+
+  static createSettingsPanel(onResult = () => {}) {
+    const viewElement = document.createElement("section");
+    viewElement.classList = ["settings-panel"];
+    viewElement.dataset.title = "Settings";
+    const linkList = document.createElement("ul");
```

Comment:
I'm not going to enforce lit for all dom' things. but still consider: 

```js
let linklist = [
 {linkTitle, callback, iconId}
]
 const viewElement = document.createElement("section");
    viewElement.classList = ["settings-panel"];
    viewElement.dataset.title = "Settings";


render(html`
<ul>
 ${linklist.map(link => html`
   <li>
    <button class="${link.iconId}" @onClick=${link.callback}> ${link.Title}</button>
   </li>
  `)}
</ul>
`, viewElement);



```

Reply:
done!